### ISSUE TITLE
File eula.html doesn't exist in the scope wrf

### ISSUE THREAD

**Issue by l29ah** [mapsme/omim#8396]
Just built maps.me on gentoo:
```
$ ./MAPS.ME
LOG TID(1) INFO   1.6929e-05 platform/string_storage_base.cpp:26 StringStorageBase() Settings path: /home/test/.config/MapsWithMe/settings.ini
LOG TID(1) WARNING 0.00022298 platform/string_storage_base.cpp:48 StringStorageBase() Loading settings: /home/test/.config/MapsWithMe/settings.ini; Read; No such file or directory
terminate called after throwing an instance of 'FileAbsentException'
  what():  FileAbsentException /home/test/tmp/omim/platform/platform.cpp:131, "File eula.html doesn't exist in the scope wrf Have been looking in:
 /
/
/home/test/.config/MapsWithMe/"
Aborted (core dumped)
```

**Comment by Zverik** [comment]
`eula.html` lies in the `data/` directory. It seems that `WritableDir` and `ResourceDir` were initialized to `/`. Please try looking into `platform/platform_linux.cpp` for the path searching algorithms.

**Comment by l29ah** [comment]
On Wed, Mar 28, 2018 at 07:39:39AM +0000, Ilya Zverev wrote:
> `eula.html` lies in the `data/` directory. It seems that `WritableDir` and `ResourceDir` were initialized to `/`. Please try looking into `platform/platform_linux.cpp` for the path searching algorithms.

Oh, i see it tries to look for the data a few `..`'s upward, even though i've built it in cwd.

-- 
()  ascii ribbon campaign - against html mail
/\  http://arc.pasp.de/   - against proprietary attachments

**Comment by l29ah** [comment]
Actually, even in case of a system-wide installation, neither cmake generates an `install` target in the Makefile, nor the docs suggest the paths to install the files from the 6GB behemoth of a build directory.

**Comment by Zverik** [comment]
Yes, we never had a task to install maps.me on a Linux system. For macOS, we have a target (see `tools/unix/build_omim.sh`). Maybe files in [debian](https://github.com/mapsme/omim/tree/master/debian) directory would help.

**Comment by l29ah** [comment]
On Thu, Mar 29, 2018 at 06:46:40AM +0000, Ilya Zverev wrote:
> Yes, we never had a task to install maps.me on a Linux system. For macOS, we have a target (see `tools/unix/build_omim.sh`). Maybe files in [debian](https://github.com/mapsme/omim/tree/master/debian) directory would help.

I couldn't find such paths either in the shell script or in the macos-specific portions of CMakeLists.txt. The debian files seem to only include the target paths, not the source ones, and seem to rely on (obsolete?) qmake-based build system.

-- 
()  ascii ribbon campaign - against html mail
/\  http://arc.pasp.de/   - against proprietary attachments

**Comment by doronbehar** [comment]
I've encountered the same issue here on Arch Linux when compiling from source. I haven't used the `build_omim.sh` script, I used the commands `cmake -DCMAKE_BUILD_TYPE=Release` and `make`.
